schema.sql

```diff
@@ -0,0 +1,63 @@
+-- A SQL schema that you can import network tester CSV into for data mining.
```

Comment:
safe to assume this is PostgreSQL dialect? Might be worth stating this in the code comment, or via the filename.